WormholeConnect-Migration-Guide-v2.5.0-to-v3.0.0.md

```diff
@@ -0,0 +1,227 @@
+  // ... all other theme properties work the same
+};
+```
+
+## Configuration Enhancements (Optional)
+
+While the core interface is unchanged, v3.0.0 adds **optional** new configuration capabilities:
+
+### 1. Enhanced Token Filtering
+
+```typescript
+// ✅ v2.5.0 - Still works in v3.0.0
+const config: WormholeConnectConfig = {
+  isTokenSupportedHandler: (token) => {
+    return token.symbol !== 'BLOCKED_TOKEN';
+  }
+};
+
+// 🆕 v3.0.0 - New optional sourceToken parameter
+const config: WormholeConnectConfig = {
+  isTokenSupportedHandler: (token, sourceToken) => {
+    // Can now filter destination tokens based on source token
+    if (sourceToken?.symbol === 'USDC' && token.symbol === 'BANNED_PAIR') {
+      return false;
+    }
+    return token.symbol !== 'BLOCKED_TOKEN';
+  }
+};
+```
+
+### 2. Route Filtering (New Feature)
+
+```typescript
+// 🆕 v3.0.0 - Optional route filtering
+const config: WormholeConnectConfig = {
+  filterRoutes: (routes) => {
+    // Hide certain routes from users
+    return routes.filter(route => route !== 'TokenBridge');
+  }
+};
+```
+
+## Import Changes
+
+### Main Import (Recommended - No Changes)
+
+```typescript
+// ✅ Works the same in both versions
+import WormholeConnect from '@wormhole-foundation/wormhole-connect';
+```
+
+### Detailed Imports (Minor Changes)
+
+If you import specific types or utilities:
+
+```typescript
+// ✅ v2.5.0
+import WormholeConnect, { 
+  WormholeConnectConfig,
+  WormholeConnectTheme,
+  MAINNET,
+  TESTNET 
+} from '@wormhole-foundation/wormhole-connect';
+
+// ✅ v3.0.0 - WormholeConnectConfig moved to config namespace
+import WormholeConnect, { 
+  config,              // 🆕 Config types now in namespace
+  WormholeConnectTheme,
+  MAINNET,
+  TESTNET 
+} from '@wormhole-foundation/wormhole-connect';
+
+// Access config types through namespace
+const myConfig: config.WormholeConnectConfig = {
+  // ... your config
+};
+```
+
+## Removed Exports
+
+The following exports were removed in v3.0.0 but were likely not used by most integrators:
+
+```typescript
+// ❌ No longer exported in v3.0.0
+import { 
+  MayanRoute,
+  MayanRouteWH, 
+  MayanRouteMCTP,
+  MayanRouteSWIFT,
+  MayanRouteSHUTTLE,
+  nttAutomaticRoute,
+  nttManualRoute,
+  nttRoutes,
+  wormholeConnectHosted,
+  HostedParameters
+} from '@wormhole-foundation/wormhole-connect';
+```
+
+## New Export Variations (Optional)
+
+v3.0.0 introduces specialized exports for different use cases:
+
+```typescript
+// Main export (recommended for most users)
+import WormholeConnect from '@wormhole-foundation/wormhole-connect';
+
+// 🆕 Specialized exports (advanced usage)
+
+// Mayan routes
+import {
+  MayanRoute,
+  MayanRouteWH,
+  MayanRouteMCTP,
+  MayanRouteSWIFT,
+} from '@wormhole-foundation/wormhole-connect/mayan';
+
+// NTT routes  
+import {
+  nttAutomaticRoute,
+  nttExecutorRoute, 
+  nttManualRoute,
+  nttRoutes,
+} from '@wormhole-foundation/wormhole-connect/ntt';
+
+// Hosted functionality
```

Comment:
This should link to the docs section for the hosted functionality.
https://wormhole.com/docs/products/connect/guides/hosted-version/
Also, this documentation is incorrect now.